### ISSUE TITLE
Missing softly{ } method

### ISSUE THREAD

**Issue by almaz010891** [allure-framework/allure-kotlin#59]
Example

```
step("step 1"){
softrly {
// do some checks
}
}
```

after migration to allure-kotlin method softly { } is not available

**Comment by viclovsky** [comment]
Yes, we have removed  soft assertion cause Allure is stands for report, not for test's logic. There are a lot of libraries which you can use. For [instance](https://github.com/bnorm/kotlin-power-assert)
